terraform/modules/log_bucket/log_bucket.tf

```diff
@@ -0,0 +1,66 @@
+locals {
+# https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-access-logs.html#access-logging-bucket-permissions
+    aws_alb_account_ids = {
```

Comment:
I don't think we need non-govcloud account ids, yeah?

Reply:
meh. I copy-pasted and formatted the whole table. No harm in it being here, as long as it's accurate.

Reply:
Let's only keep the govcloud ones, just to be safe.

Reply:
I don't see added safety in removing them, since they're keyed off of the aws region we're in.

Reply:
Under the doc page you linked, this was the first requirement:

> The bucket must be located in the same region as the load balancer.

We cannot deploy to any of the other regions and since we can't deploy to those regions, we can't use it, which means we're putting extraneous account principals into our bucket access policy.

Reply:
that's *exactly* why I think it's valuable to keep these here. 
Our contingency plan includes the possibility of rolling to a non-govcloud AWS region.

Again, we're indexing into this map based on the region we're deploying to. The only way we'll access the non-govcloud regions here is if the rest of the deployment is going to a non-govcloud region. This module should not be the one that decides which regions are/are not allowed.

Reply:
You're right, I missed it:

```
aws_alb_account_ids[var.aws_partition]
```